### PR TITLE
update index page to show first 50 packages with detailed info

### PR DESCRIPTION
**Pull request by zivillian**
I've updated the default.aspx to include some details from package feed. It's currently limited to the first 50 packages (which hopefully will work for most private feeds).

The design is based on Bootstrap and using some assets from public CDNs (which may be a problem in some environments)

The page was tested using ~ 10 packages with ~ 30 versions.

### CODE REVIEW

**Comment by ericnewton76** [comment]
whelp... something about your line endings makes the diff look completely different, can you try to fix that?

I was gonna make an update to Default.aspx in my fork that this commit would conflict with

**Comment by ericnewton76** [comment]
As a review, realistically I would say, keep it simple... dont bring in a bunch of CSS/JS from everywhere, when a simple html table will do just fine.   This will probably greatly increase your chances of getting it accepted.

File: src/NuGet.Server/Default.aspx [hunk 1]
```diff
@@ -1,49 +1,247 @@
...
-</body>
-</html>
+﻿<%@ Page Language="C#" %>
+<%@ Import Namespace="NuGet" %>
+<%@ Import Namespace="NuGet.Server" %>
+<%@ Import Namespace="NuGet.Server.DataServices" %>
+<%@ Import Namespace="NuGet.Server.Infrastructure" %>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<%
+    var service = new Packages();
```
Comment (ericnewton76) [inline src/NuGet.Server/Default.aspx]: Error: The type or namespace name 'Packages' could not be found

File: src/NuGet.Server/Default.aspx [hunk 2]
```diff
@@ -1,49 +1,247 @@
...
+                {
+                    var first = package.First();%>
+                 <li><a href="#<%= package.Key.Id.Replace(' ', '_')%>"><%= first.Title %></a></li>     
+              <%}%>
+          </ul>
+        </div>
+        <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
+            <h1>ahd Nuget Server</h1>
+            <div>
+                <h2>You are running NuGet.Server v<%= typeof(NuGet.Server.DataServices.ODataPackage).Assembly.GetName().Version %></h2>
```
Comment (ericnewton76) [inline src/NuGet.Server/Default.aspx]: Error: The type or namespace name 'ODataPackage' does not exist in the namespace 'NuGet.Server.DataServices' (are you missing an assembly reference?)

**Comment by dodu2014** [comment]
Hello, I ask you a question, you have 3.1.2 test. I used here is, not through the NPE to publish my packages. The error is: Not Found. do you encountered such a situation, or where I set it right.

**Comment by chenriksson** [comment]
@zivillian @skofman1 Closing, as it looks like there was unaddressed feedback and the PR is now based on an old version of NuGet.Server (2.14.0).

If you would still like to contribute, I encourage you to update the PR to the latest code and resubmit.

**Pull request closed without merging**